cmake/Modules/FindSphinx.cmake

```
@@ -8,7 +8,11 @@
 find_package(PkgConfig QUIET)
 pkg_check_modules(PC_SPHINX sphinx-build)
 
-find_package(PythonInterp)
+if(CMAKE_VERSION VERSION_LESS 3.12.0)
+  find_package(PythonInterp)
+else()
+  find_package(Python)
+endif()
 
 if(PYTHONINTERP_FOUND)
```

```suggestion
if(Python_Interpreter_FOUND)
```
per https://cmake.org/cmake/help/latest/module/FindPython.html#result-variables

---

Or put them resepectively in each if else block with their find_pacakge call :smile:

---

Right. I understand that the variables that new package outputs are different than old package. Thanks

---

I will correct this one as well.
* find_package(PythonInterp) -> PYTHON_EXECUTABLE
* find_package(Python) -> Python_EXECUTABLE